src/with-content-rect.js

```diff
@@ -63,12 +63,11 @@ function withContentRect(types) {
         this._animationFrameID = window.requestAnimationFrame(() => {
           if (this._resizeObserver !== null) {
             this.setState({ contentRect })
+            if (typeof this.props.onResize === 'function') {
+              this.props.onResize(contentRect)
```

Comment:
This fixed the issue locally for me and seems to be what other solutions have done as well: https://github.com/maslianok/react-resize-detector/commit/9625f67a48ee76a788b316c3fede083a419bd661#diff-1fdf421c05c1140f6d71444ea2b27638R82

Reply:
@souporserious lemme know if this makes sense. Thanks!

Reply:
Makes sense! Thanks for the PR! 🙏